zktest/src/archive/test2/F85-ZK-1148-2.zul

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+F85-ZK-1148-2.zul
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Mon Jul 02 16:06:11 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+
+-->
+
+<zk>
+	<!--<window id="win"-->
+			<!--viewModel="@id('vm')@init('org.zkoss.zktest.test2.F85_ZK_1148_IncludeVM')">-->
+		<!--<button label="show vm attribute"-->
+				<!--onClick='System.out.println("vm = " + win.getAttributes().get(org.zkoss.bind.sys.BinderCtrl.VM));'/>-->
+	<!--</window>-->
```

Comment:
Remove these comments if they are not used anymore.

Reply:
ok